src/screens/Signup.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+import { useState } from 'react';
+
+export default function SignUpForm() {
```

Comment:
When I load the screen I'm getting this: 

![PNG image](https://user-images.githubusercontent.com/63211322/198196753-d253583d-adda-4d20-8040-b0e21a462c1c.png)

Not sure if this is just on my end, but I'm gonna look into it some more